toolkit/tools/pkg/imagecustomizerlib/customizeoverlays.go

```diff
@@ -29,6 +32,13 @@ func enableOverlays(overlays *[]imagecustomizerapi.Overlay, selinuxMode imagecus
 
 	logger.Log.Infof("Enable filesystem overlays")
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "enable_overlays")
+	span.SetAttributes(
+		attribute.Int("overlays_count", len(*overlays)),
+		attribute.String("selinux_mode", string(selinuxMode)),
```

Comment:
The SELinux mode is likely logged elsewhere. So, you probably don’t need to log it here as well.